hack/update-vendor.sh

```diff
@@ -21,6 +21,7 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 source "${KUBE_ROOT}/hack/ensure-go.sh"
 
+export GOPROXY=https://proxy.golang.org
```

Comment:
```suggestion
export GOPROXY=${GOPROXY:-https://proxy.golang.org}
```

Reply:
```suggestion
GOPROXY=$(go env GOPROXY)
export GOPROXY="${GOPROXY:-https://proxy.golang.org}"
```

Reply:
Is it worth allowing the user to take priority if they want to specify the value?
```
export GOPROXY="${GOPROXY:-$(go env GOPROXY)}"
export GOPROXY="${GOPROXY:-https://proxy.golang.org}"
```

Reply:
`go env` already handles that?

Reply:
Didn't realize that - thanks!

Reply:
NP, this should configure with: an explicitly set env on the command line OR ELSE the global default go setting OR ELSE if neither of those are set (default for go 1.12) then it will set the hard coded proxy 🙂